app/main.py

```diff
@@ -1,3 +1,50 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        return Distance(self.km + other.km) \
+            if isinstance(other, Distance) else Distance(self.km + other)
```

Comment:
i think it will look better
generally try to avoid `/`
```
        return (
            Distance(self.km + other.km)
            if isinstance(other, Distance)
            else Distance(self.km + other)
        )
```

also please fix this in other parts of the code

Reply:
i fixed the mistakes